srcpkgs/scons/template

```diff
@@ -1,18 +1,30 @@
 # Template file for 'scons'
 pkgname=scons
-version=4.0.1
-revision=3
-wrksrc="SCons-${version}"
-build_style=python3-module
-hostmakedepends="python3 python3-setuptools"
+version=4.4.0
+revision=1
+#build_style=python3-module
+hostmakedepends="python3 python3-setuptools python3-psutil python3-wheel python3-pytest tar"
 depends="python3"
 short_desc="Software construction tool"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Wilson Birney <wpb@360scada.com>"
 license="MIT"
 homepage="https://www.scons.org/"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
-checksum=722ed104b5c624ecdc89bd4e02b094d2b14d99d47b5d0501961e47f579a2007c
+distfiles="https://github.com/SCons/scons/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz"
```

Comment:
Please don't `>${pkgname}-${version}.tar.gz`

Reply:
Anyway, scons homepage still points to sourceforge.

Reply:
If you prefer GitHub, use this link: https://github.com/SCons/scons/releases/download/4.4.0/SCons-4.4.0.tar.gz

Reply:
I reverted to the sourceforge link in my latest local copy. Ill drop the >${pkgname}-${version}.tar.gz as well

thanks!